SDL2_netConfig.cmake.in

```diff
@@ -0,0 +1,26 @@
+set(SDL2_net_FOUND ON)
+
+set(SDL2NET_SDL2_REQUIRED_VERSION  @SDL_REQUIRED_VERSION@)
+
+include(CMakeFindDependencyMacro)
+
+#FIXME: can't add SDL2NET_SDL2_REQUIRED_VERSION since not all SDL2 installs ship SDL2ConfigVersion.cmake
+if(EXISTS "${CMAKE_CURRENT_LIST_DIR}/SDL2_image-shared-targets.cmake")
+    if(NOT TARGET SDL2::SDL2)
+        find_dependency(SDL2)
```

Comment:
We are in the process of adding cmake component support to SDL2.
In the future the following can be done then:
```cmake
        find_dependency(SDL2 REQUIRED COMPONENTS SDL2 )
```
and 
```cmake
        find_dependency(SDL2 REQUIRED COMPONENTS SDL2-static)
```
There will also be a `SDL2main` and `SDL2test` component.
So in the future, there will be no more need for any `FindSDL2xxx.cmake` modules.